### ISSUE TITLE
Predicting lymph node metastasis from primary tumor histology and clinicopathologic factors in colorectal cancer using deep learning

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#23]
## TL;DR

Presence of lymph node metastasis (LNM) influences prognosis and clinical decision-making in colorectal cancer. However, detection of LNM is variable and depends on a number of external factors. Deep learning has shown success in computational pathology, but has struggled to boost performance when combined with known predictors.

### Paper Link

https://www.nature.com/articles/s43856-023-00282-0#citeas

### Author/Institution

Justin D. Krogue (Google Health)

## Overview

In this work, we identify machine-learned histopathologic features of primary resection specimens that predict the presence of LNM in CRC. Notably, these features provide independent signal relative to known clinicopathologic variables. While the boost in predictive performance achieved via the addition of ML features to our baseline model is modest, it does indicate that there is additional signal for LNM prediction when combined with what is currently known and used (e.g., T-stage, grade, lymphovascular invasion, venous invasion, etc), representing an opportunity for future research to further understand the features and biology associated with LNM. Without implying that this model is immediately applicable for clinical use, there are at least two important clinical decisions related to this type of prediction. First, for prognostic risk stratification to help identify high risk patients with Stage II cancer who may benefit from adjuvant chemotherapy[2](https://www.nature.com/articles/s43856-023-00282-0#ref-CR2),[3](https://www.nature.com/articles/s43856-023-00282-0#ref-CR3). Second, for risk assessment in endoscopic resection of apparent T1 cancer where there lymph node sampling is typically not performed but risk of metastasis is still 7–15%.

Evaluation of this model as a risk stratification tool (in the DSS analysis) is limited as this is a retrospective study, and treatment pathways present an important confounding factor that is difficult to control for, including potential differences in neoadjuvant and adjuvant therapy. Though treatment guidelines within stage II and within stage III colorectal cancer cohorts are fairly uniform, at least some variability in treatment likely exists in the real world. 

## Methods

colorectal cancer. survival analysis

## Cite

Krogue, J.D., Azizi, S., Tan, F. et al. Predicting lymph node metastasis from primary tumor histology and clinicopathologic factors in colorectal cancer using deep learning. Commun Med 3, 59 (2023). https://doi.org/10.1038/s43856-023-00282-0
